recipes/rtmidi/all/conandata.yml

```
@@ -1,9 +1,9 @@
 sources:
   "6.0.0":
-    url: "https://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-6.0.0.tar.gz"
+    url: "http://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-6.0.0.tar.gz"
```

```suggestion
    url:
      - "https://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-6.0.0.tar.gz"
      - "http://www.music.mcgill.ca/~gary/rtmidi/release/rtmidi-6.0.0.tar.gz"
```
I would add http as a fall-back option instead of dropping https entirely.

---

Just changed it for all the URLs to contain both URLs